src/scss/components/search/_index.scss

```diff
@@ -223,5 +235,6 @@ hx-search-control:not([hx-defined]) > button {
 
   &:hover {
     background-color: $cyan-50;
+    background-color: var(--hxSearch-hxSearchSuggestion-hover-bgcolor, $cyan-50);
```

Comment:
Please use updated  `v2.0` colors to replace `$cyan-XXX`.

Reply:
Changes have been done, Please review